app/models/OkcBlotter/offense.rb

```diff
@@ -1,4 +1,8 @@
 class OkcBlotter::Offense < ApplicationRecord
-  belongs_to :booking, class_name: 'OkcBlotter::Booking', foreign_key: 'bookings_id'
-  validates :type, :charge, presence: true
+  belongs_to :booking, class_name: 'OkcBlotter::Booking'
+
+  class ::ActiveRecord::Base
+    # disable STI to allow columns named "type"
+    self.inheritance_column = :_type_disabled
+  end
```

Comment:
we have a field called "type" which is used in rails for single table inheritance so we have to disable that feature to keep the column name

Reply:
yup